addons/sprite_painter/src/editing_tools/tool_brush.gd

```diff
@@ -166,10 +175,13 @@ func stroke(stroke_start, stroke_end, pressure):
 				pressure
 			)
 
-
+#########################################################################################
```

Comment:
What is this?

Reply:
Oh, I just marked the place with a comment for myself.
 Sorry for the little dirt, I often don’t delete comments that I use during development.